response_operations_ui/templates/home.html

```diff
@@ -10,6 +10,7 @@
       <ul class="u-pt-l">
         <li><a name="surveys-link" href="/surveys/">View list of business surveys</a></li>
         <li><a name="messages-link" href="/messages/">View messages</a></li>
+        <li><a name="create-message-link" href="/messages/create-message?ru_details=survey%3DBRES%2B2017%26ru_ref%3D49900000280%26business%3DBolts%2B%2526%2BRachets%2BLtd%26to%3DJacky%2BTurner%26to_uuid%3Df62dfda8-73b0-4e0e-97cf-1b06327a6712%26to_ru_id%3Dc614e64e-d981-4eba-b016-d9822f09a4fb">Create message link</a>
```

Comment:
I don't know that the parameters need to be there. Are they there for a reason?

Reply:
This page should normally only be accessed through a page that passes it these parameters, it doesn't work without them